src/server/pbsd_init.c

```diff
@@ -334,6 +344,15 @@ init_server_attrs()
 				&server.sv_attr[(int)SVR_ATR_DefaultChunk],
 				(void *)&server, ATR_ACTION_NEW);
 		}
+		presc = add_resource_entry(&server.sv_attr[SRV_ATR_resource_deflt], prdef);
+		if (presc) {
+			presc->rs_value.at_val.at_long = 1;
+			presc->rs_value.at_flags =
+				ATR_VFLAG_DEFLT|ATR_VFLAG_SET|ATR_VFLAG_MODCACHE;
+			server.sv_attr[(int)SRV_ATR_resource_deflt].at_flags =
+				ATR_VFLAG_DEFLT|ATR_VFLAG_SET|ATR_VFLAG_MODCACHE;
+		}
```

Comment:
I don't think we need to call action function here, it just check if the value of resource is +ve or not. Since we are setting the default value here, it is ok not to call it.

Reply:
well that's what the action function does today, I think it's safer to call it, but I'll leave the decision to you.

Reply:
I spoke to Bhroam about this, we both think that it is not needed to call action function unless it is doing more than just validating the value. In this case, since it is just checking for negative values, I'd rather leave it as is.